bothub/api/v2/evaluate/filters.py

```diff
@@ -110,6 +110,12 @@ class Meta:
         help_text=_("Filter for repository cross_validation results."),
     )
 
+    type = filters.CharFilter(
+        field_name="type", 
```

Comment:
this line contains trailing whitespace in:
`field_name="type", `
for more details: [](https://www.flake8rules.com/rules/W291.html)